anyway/models.py

```diff
@@ -791,6 +792,68 @@ def is_anonymous(self):
     def get_id(self):
         return self.id
 
+class WazeAllerts(Base):
```

Comment:
Waze alerts table exists (See class WazeAlert in this file)
What there's a need in another one?